src/data/balance/signature-species.ts

```diff
@@ -11,87 +11,87 @@ export type SignatureSpecies = {
+  FALKNER: [Species.PIDGEY, Species.HOOTHOOT, Species.NATU, Species.MURKROW],
+  BUGSY: [Species.SCYTHER, Species.SHUCKLE, Species.YANMA, [Species.PINSIR, Species.HERACROSS]],
+  WHITNEY: [Species.MILTANK, Species.IGGLYBUFF, Species.AIPOM, [Species.GIRAFARIG, Species.STANTLER]],
+  MORTY: [Species.GASTLY, Species.MISDREAVUS, Species.DUSKULL, Species.SABLEYE],
+  CHUCK: [Species.POLIWRATH, Species.TYROGUE, Species.MANKEY, Species.MACHOP],
+  JASMINE: [Species.STEELIX, Species.MAGNEMITE, Species.PINECO, Species.SKARMORY],
+  PRYCE: [Species.SWINUB, Species.SEEL, Species.SHELLDER, Species.SNEASEL],
+  CLAIR: [Species.HORSEA, Species.DRATINI, Species.GYARADOS, Species.DRUDDIGON], // Tera Dragon Gyarados
   // Gym Leaders- Hoenn
-  ROXANNE: [Species.GEODUDE, Species.NOSEPASS],
-  BRAWLY: [Species.MACHOP, Species.MAKUHITA],
-  WATTSON: [Species.MAGNEMITE, Species.VOLTORB, Species.ELECTRIKE],
-  FLANNERY: [Species.SLUGMA, Species.TORKOAL, Species.NUMEL],
-  NORMAN: [Species.SLAKOTH, Species.SPINDA, Species.ZIGZAGOON, Species.KECLEON],
+  ROXANNE: [Species.NOSEPASS, Species.GEODUDE, [Species.ANORITH, Species.LILEEP], Species.ARON],
+  BRAWLY: [Species.MAKUHITA, Species.MACHOP, Species.MEDITITE, Species.SHROOMISH],
+  WATTSON: [Species.ELECTRIKE, Species.VOLTORB, Species.MAGNEMITE, [Species.PLUSLE, Species.MINUN]],
+  FLANNERY: [Species.TORKOAL, Species.SLUGMA, Species.NUMEL, Species.HOUNDOUR],
+  NORMAN: [Species.SLAKOTH, Species.SPINDA, Species.WHISMUR, Species.KECLEON],
   WINONA: [Species.SWABLU, Species.WINGULL, Species.TROPIUS, Species.SKARMORY],
-  TATE: [Species.SOLROCK, Species.NATU, Species.CHIMECHO, Species.GALLADE],
+  TATE: [Species.SOLROCK, Species.NATU, Species.CHINGLING, Species.GALLADE],
   LIZA: [Species.LUNATONE, Species.SPOINK, Species.BALTOY, Species.GARDEVOIR],
-  JUAN: [Species.HORSEA, Species.BARBOACH, Species.SPHEAL, Species.RELICANTH],
+  JUAN: [Species.HORSEA, Species.SPHEAL, Species.BARBOACH, Species.CORPHISH],
   // Gym Leaders- Sinnoh
-  ROARK: [Species.CRANIDOS, Species.LARVITAR, Species.GEODUDE],
-  GARDENIA: [Species.ROSELIA, Species.TANGELA, Species.TURTWIG],
-  MAYLENE: [Species.LUCARIO, Species.MEDITITE, Species.CHIMCHAR],
+  ROARK: [Species.CRANIDOS, Species.GEODUDE, Species.NOSEPASS, Species.LARVITAR],
+  GARDENIA: [Species.BUDEW, Species.CHERUBI, Species.TURTWIG, Species.LEAFEON],
+  MAYLENE: [Species.RIOLU, Species.MEDITITE, Species.CHIMCHAR, Species.CROAGUNK],
   CRASHER_WAKE: [Species.BUIZEL, Species.WOOPER, Species.PIPLUP, Species.MAGIKARP],
-  FANTINA: [Species.MISDREAVUS, Species.DRIFLOON, Species.SPIRITOMB],
-  BYRON: [Species.SHIELDON, Species.BRONZOR, Species.AGGRON],
-  CANDICE: [Species.SNEASEL, Species.SNOVER, Species.SNORUNT],
-  VOLKNER: [Species.SHINX, Species.CHINCHOU, Species.ROTOM],
+  FANTINA: [Species.MISDREAVUS, Species.DRIFLOON, Species.DUSKULL, Species.SPIRITOMB],
+  BYRON: [Species.SHIELDON, Species.BRONZOR, Species.SKARMORY, Species.ARON],
```

Comment:
```suggestion
  BYRON: [Species.SHIELDON, Species.MAGNEMITE, Species.BRONZOR, Species.ARON],
```
He uses Magneton/Magnezone pretty consistently throughout his teams, even replacing Bronzor with it in Platinum and the BDSP rematch.

Reply:
and he uses Skarmory as much as the Magnemite line, there's way too many Magnemite, better to cut one or two where we can here

Reply:
wouldn’t mind retaining it (esp with Magnezone being a Gen 4 addition) I believe the change was only due to slightly cutting down on the abundance of Magnemite

Reply:
Magnemite is on all his teams in Platinum and B2W2, and his rematch team in BDSP. He only doesn't have it in DP. It also parallels Roark using Nosepass.
There's also Steelix but I know that's already on a lot of GLs by now.

Reply:
Unanimous against in the discord